src/main/webapp/TempInfoWindow.js

```diff
@@ -60,72 +60,38 @@ class TempInfoWindow extends InfoWindowTemplate {
    * Returns the HTML of the right section of the information window
    */
   makeRightColumn_() {
-    let rightCol = document.createElement("div");
-    rightCol.classList.add(InfoWindowTemplate.RIGHT_COLUMN);
-    rightCol.appendChild(this.makeContentForm_());
-    return rightCol;
-  }
-
-  /**
-   * @Private
-   * Returns the HTML for the input form for the marker description
-   */
-  makeContentForm_() {
-    let form = document.createElement("div");
+    let rightCol = myMap.makeEl("div", InfoWindowTemplate.RIGHT_COLUMN);
 
-    //get the marker that is currently being edited to populate the form
     let editedMarker = myMap.editingPermMarker();
+    let titleText = editedMarker? editedMarker.getTitle(): null;
+    let bodyText = editedMarker? editedMarker.getBody(): null;
 
-    form.appendChild(this.makeTitleInput_(editedMarker));
-    form.appendChild(this.makeBodyInput_(editedMarker));
-    form.appendChild(this.makeSubmitBtn_());
-    return form;
-  }
-
-  /**
-   * @Private
-   * Builds the HTML for the textarea to input the title for the marker
-   * @param {PermMarker} editedMarker perm marker that is being edited
-   */
-  makeTitleInput_(editedMarker) {
-    let titleInput = document.createElement("textarea");
-    titleInput.id = TempInfoWindow.TITLE_INPUT;
-    titleInput.placeholder = TempInfoWindow.DEFAULT_TITLE;
+    let titleInput = this.makeForm_(TempInfoWindow.TITLE_INPUT,
+        TempInfoWindow.DEFAULT_TITLE, titleText);
+    rightCol.appendChild(titleInput);
 
-    if (editedMarker) {
-      let title = editedMarker.getTitle();
-      titleInput.innerHTML = title? title: "";
-    }
+    let bodyInput = this.makeForm_(TempInfoWindow.BODY_INPUT,
+        TempInfoWindow.DEFAULT_BODY, bodyText);
```

Comment:
same concern here.

Reply:
The reasoning for titleInput is applicable here as well.